src/routes/+page.svelte

```diff
@@ -1,18 +1,47 @@
 <script lang="ts">
 	import Card from '$lib/components/Card.svelte';
 	import type { PageData } from './$types';
-	import SearchBar from './SearchBar.svelte';
 
 	export let data: PageData;
+	let searchQuery = '';
+
+	const onSearch = (event: Event) => {
+		event.preventDefault();
+		const query = new URLSearchParams({ query: searchQuery }).toString();
+		window.location.href = `?${query}`;
```

Comment:
If using the built in `goto()` function this would make it be client side rendered. So state would not change everytime someone searches.